__tests__/integration/one-app.spec.js

```diff
@@ -544,6 +544,26 @@ describe('Tests that require Docker setup', () => {
       });
     });
 
+    test('app calls loadModuleData to run async requests using Tenant provided fetchClient', async () => {
```

Comment:
Moved this up to the `Docker required` section of the integration tests as this test will never pass when running against a "real" environment because the API servers are not deployed.